miden-tx/src/tests/kernel_tests/test_account.rs

```diff
@@ -484,9 +484,15 @@ fn test_storage_offset() {
     ];
     let code = AccountCode::from_parts(code.mast().clone(), procedures_with_offsets.clone());
 
+    // this manual setting of storage slot at location 2 is required because
+    // the default behaviour of [AccountBuilder::new] initialises a map at
+    // location 2 of storage where we need a value.
+    let storage_slot = StorageSlot::Value(Word::default());
```

Comment:
This should probably just be removed from `AccountBuilder::new()`. I guess it was there for the (now ignored?) storage map tests?